Modules/System/Upgrade Tags/src/UpgradeTag.Codeunit.al

```diff
@@ -48,6 +48,25 @@ codeunit 9999 "Upgrade Tag"
         UpgradeTagImpl.SetUpgradeTag(NewTag);
     end;
 
+    /// <summary>
+    /// Sets the upgrade tag to skipped.
+    /// </summary>
+    /// <param name="ExistingTag">Tag code to set the skipped field</param>
+    procedure SetSkippedUpgrade(ExistingTag: Code[250])
```

Comment:
Consider adding GetSkippedUpgrade(ExistingTag: Code) : Boolean and GetSkippedUpgrade(ExistingTag: Code, CompanyName: Text): Boolean) so you can check..

Reply:
Added boolean